src/main/java/mods/railcraft/common/core/RailcraftConfig.java

```diff
@@ -335,6 +335,7 @@ private static void loadRecipeOption() {
 
         loadRecipeProperty("minecraft.furnace", "creosote", false, "change to '{t}=true' to add smelting recipes for Creosote Oil to the vanilla furnace");
         loadRecipeProperty("railcraft.track", "useAltRecipes", false, "change to '{t}=true' to use track recipes more similar to vanilla minecraft");
+        loadRecipeProperty("railcraft.bronze", "enableAltBronze", false, "change to '{t}=true' to enable a recipe to craft Bronze Ingots from Tin and Copper Ingots");
```

Comment:
That's intended to be used. I haven't figured out where to put it (being the recipe) yet. Remember this is a WIP for a reason. Need to ask CJ about that.